packages/graphile-build-pg/src/plugins/makeProcField.js

```diff
@@ -190,10 +222,41 @@ export default function makeProcField(
       fieldScope.pgFieldIntrospectionTable = returnTypeTable;
       payloadTypeScope.pgIntrospectionTable = returnTypeTable;
     }
+  } else if (isRecordLike) {
+    const RecordType = getTypeByName(
+      inflection.functionReturnsRecordType(proc)
+    );
```

Comment:
Handle `RecordType` not found issue (just don't make the field)

Reply:
See 92dbc7e; makeProcField is expected to return a field spec, so it has to throw instead of returning silently. I didn't want to get into refactoring the calls to makeProcField unless you think that's necessary.

Reply:
No that's fine, I think many are wrapped with swallowError anyway. I just want to ensure that if someone deliberately skips the record plugin that they will still be able to boot their PostGraphile instance even if it has INOUT / RETURNS TABLE functions.